fv/containers/containers.go

```diff
@@ -243,6 +288,7 @@ func (c *Container) CanConnectTo(ip, port, protocol string) bool {
 
 func RunEtcd() *Container {
 	return Run("etcd",
+		RunOpts{true},
```

Comment:
Please can you spell out the field name here an below?  If we add another option field then this will break.